moveit_ros/moveit_servo/test/servo_launch_test_common.hpp

```diff
@@ -113,7 +113,7 @@ class ServoFixture : public ::testing::Test
     // Otherwise the Servo is still running when another test starts...
     if (!client_servo_stop_)
     {
-      client_servo_stop_->async_send_request(std::make_shared<std_srvs::srv::Trigger::Request>());
+      stop();
```

Comment:
I'm not exactly sure why the previous command was causing null pointers, but this seems to compile and passes tests.

Reply:
Was there a path where `client_servo_stop_` wasn't valid?